### PR TITLE
new Inventory Page, logic to route and add inventory to DB.

### PR DESCRIPTION
**Pull request by swong452**
Create page for adding new Inventory.
No Checking logic yet on SKU but if auto populate may not need this check ? 

Add routing logic, so from inventory page; click the NewInventory button will take user to the AddNew Inventory FORM.

Add logic so user can add new product, and will be shown in DB product table.

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -23,5 +24,6 @@
 
         {{!-- this file is for the button handling in the NewOrder modal --}}
         <script src="/js/newOrder.js"></script>
```
Comment (tabouzeid) [inline views/layouts/main.handlebars:26]: These should not be in the main... they should be in the page they’re clicked

Comment (tabouzeid) [inline views/layouts/main.handlebars:26]: These should not be in the main... they should be in the page they’re clicked

**Review by tabouzeid — CHANGES_REQUESTED** [review]

File: views/layouts/main.handlebars (hunk 1 above)
Comment (swong452) [reply views/layouts/main.handlebars:26]: Hi Taher, if i understand correctly, the PR comment mentioned 'newOrder.js' should not be in the main.handlebars; but instead, in the associated handlebars (i.e: newOrders.handlebars)
I take a look at wk 13 MVC , Day 3, class activities #17, the CaApp example. Below; it shows the cat.js is not defined under cat.handlebars - but instead; within main.handlebars.....
I originally thought the same way you did, but then, the <head> is only defined in main.handlebars; so may be that's why in catApp ex,
*.js are defined under main.handlebars ?....what do you think...

Comment (swong452) [reply views/layouts/main.handlebars:26]: tested in newInventory.js and not in main.handlebar; page did not come up.
agree so far to leave it on main.handlebars until further testing.

**Comment by swong452** [comment]
proceed to merge base on agreement

**Review by tabouzeid — APPROVED** [review]

**Pull request merged**